backend/api/utils.py

```diff
@@ -243,4 +243,40 @@ def handle_uploaded_file(app_name, file, filename):
 
     with open(cumulative + '/' + filename, 'wb+') as destination:
         for chunk in file.chunks():
-            destination.write(chunk)
\ No newline at end of file
+            destination.write(chunk)
+
+
+def annotation_json(annotation):
```

Comment:
We need to call other json funcs below from here so we have a json response fit to w3c standard

Reply:
Or ignore the comment above and remove this function as we have ```def annotation_object(self):``` func in model

Reply:
Note that annotation_object(self) would not work the same way without annotation_json(annotation), and there would be more redundancy